src/sample/index.ts

```diff
@@ -18,15 +19,21 @@ export class ExampleStack extends cdk.Stack {
 
     // user's lambda function
```

Comment:
Nit: I know this was already there, but I'm not sure this comment ("user's lambda function") makes sense given that there is more than one Lambda function defined in this sample app. I would 🔪 it.